flow/embedded_views.h

```diff
@@ -205,6 +206,15 @@ class ExternalViewEmbedder {
       int view_id,
       std::unique_ptr<EmbeddedViewParams> params) = 0;
 
+  // This needs to get called after |Preroll| finishes on the layer tree.
+  // Returns false if the frame needs to be processed again, this is after
+  // it does any requisite tasks needed to bring itseld to a valid state.
+  // Returns true if the view embedder is already in a valid state.
+  virtual bool PostPrerollAction(
```

Comment:
nit: the meaning of the return value may not be immediately clear on the call site from the method's name, consider returning an enum.

Reply:
Done